packages/form/src/SelectRow.js

```diff
@@ -135,29 +147,30 @@ class SelectRow extends PureComponent {
                 closable={CLOSABLE_CONFIG}
                 onClose={this.handlePopoverClose}>
                 <SelectList
-                    values={this.state.cachedValues}
+                    value={this.state.cachedValue}
                     onChange={this.handleSelectChange}
                     {...selectListProps} />
             </Popover>
         );
     }
 
     renderRowValuesAside() {
-        const { multiple, asideAll, asideNone, asideSeparator } = this.props;
-        const { cachedValues, valueLabelMap } = this.state;
+        const { multiple, asideAllLabel, asideNoneLabel, asideSeparator } = this.props;
+        const { cachedValue, valueLabelMap } = this.state;
 
-        if (cachedValues.length === 0) {
-            return <span className={BEM.placeholder.toString()}>{asideNone}</span>;
+        if (multiple && cachedValue.length === 0) {
```

Comment:
What if it's single-choice mode and it does not have a value?

Reply:
nice catch!